deployability/modules/testing/models.py

```diff
@@ -14,29 +14,27 @@ class ExtraVars(BaseModel):
 class InputPayload(ExtraVars):
     """Input payload for testing module."""
     tests: list[str]
-    inventory: Path
-    dependencies: list[str] = []
+    targets: list[str]
+    dependencies: list[str]
     cleanup: bool = True
 
+
     @field_validator('tests', mode='before')
     def validate_tests(cls, value) -> list[str]:
         """Validate tests names."""
         if type(value) is str:
             value = value.split(',')
+
         return value
 
-    @field_validator('inventory', mode='before')
-    def validate_inventory(cls, value) -> Path:
-        """Validate inventory path."""
-        if not Path(value).exists():
-            raise ValueError(f'Inventory file "{value}" does not exist')
-        return Path(value)
+    @field_validator('targets', mode='before')
+    def validate_targets(cls, values) -> list:
+        """Validate required fields."""
+
+        return values
 
     @model_validator(mode='before')
-    def validate_dependencies(cls, values) -> dict:
+    def validate_dependencies(cls, values) -> list:
         """Validate required fields."""
-        if isinstance(values['dependencies'], str):
-            values['dependencies'] = values['dependencies'].split(',')
-        if values.get('component') == 'agent' and not values.get('dependencies'):
-            raise ValueError('dependencies are required when component is agent')
+
         return values
```

Comment:
As the previous, this validator must be deleted, as its not adding any validation or value casting.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5112/commits/dd818b765d551ab2c6431656986ec60cea05225c